library/manageiq_policy.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+
+import os
+from ansible.module_utils.basic import *
```

Comment:
@dkorn,

I was specifically referring to the ansibe import code.  That allows you to import your library in python and work with it.